batch/batches/Archive/KAsyncLiveEntryArchive.class.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+/**
+ * @package Scheduler
+ * @subpackage Copy
+ */
+
+class KAsyncLiveEntryArchive extends KJobHandlerWorker
+{
+    const MAX_CUE_POINTS_PER_PAGE = 100;
+
+    /* (non-PHPdoc)
+     * @see KBatchBase::getType()
+     */
+    public static function getType()
+    {
+        return KalturaBatchJobType::LIVE_ENTRY_ARCHIVE;
+    }
+
+    /**
+     * (non-PHPdoc)
+     * @see KBatchBase::getJobType()
+     */
+    protected function getJobType()
+    {
+        return KalturaBatchJobType::LIVE_ENTRY_ARCHIVE;
+    }
+
+    /**
+     * @param KalturaBatchJob $job
+     * @return KalturaBatchJob
+     */
+    protected function exec(KalturaBatchJob $job)
+    {
+        KalturaLog::info("Starting live entry archive! big news!");

```

Comment:
no need for this log message, every batch already has an informative "starting" log line